### DISCUSSION TITLE
Only showing date on a Chart with a Date Hierarchy

### DISCUSSION THREAD

**Discussion by ncosgrov58** [atoti/atoti#783] [Q&A]
I am creating a plot where the the day element is part of a hierarchy created with the create_date_hierarchy function and is used on the x-axis.  While the results are correct, as you can see the reslting graph is messy.  Is there a way to just plot on date without the Year, Quarter, Month annotation?  (note that those level are relative in other dashboard widgets, just not this one)

  
![Untitled](https://user-images.githubusercontent.com/57536496/236325560-552db839-aca0-4d55-95fd-ad264a066d04.png)

**Comment by tibdex** [comment]
Hi, have you tried:

```python
h["Date"] = [column_that_was_passed_to_create_date_hierarchy]
```

and then use that **Date** hierarchy in your chart instead of the multilevel date hierarchy?

**Comment by ncosgrov58** [comment]
Yes, I realize I could do that, but its a bit messy now having two hierarchical representations of the same thing, was hoping to just have one concept of date, not date in a hierarchy and date on its own/
